python/metatensor_torch/metatensor/torch/atomistic/_extensions.py

```diff
@@ -32,8 +32,10 @@ def _featomic_deps_path():
         for prefix in site_packages:
             # find where featomic is and look inside the `featomic_torch.libs` directory
             if os.path.exists(os.path.join(prefix, "featomic")):
-                correct_prefix = prefix
-                libs_list += os.listdir(os.path.join(prefix, "featomic_torch.libs"))
+                libs_dir = os.path.join(prefix, "featomic_torch.libs")
+                if os.path.exists(libs_dir):
+                    correct_prefix = prefix
```

Comment:
Are you sure that this will not break further below 

```python
        gomp_dependency = [
            os.path.join(correct_prefix, "featomic_torch.libs", libgomp_path)
        ]
```
here you use `correct_prefix` but it might not be defined because you only define this within this condition

Reply:
Good catch, thanks